docs/manual/inference.tex

```diff
@@ -229,6 +228,31 @@
 
 You may need to wait a few minutes for the command to complete.
 
+\subsectionAndLabel{Whole-program inference results}{wpi-results}
+
+The result of invoking \<wpi.sh> are log files stored in the \<dljc-out/> folder
+of the target project.
+The \<dljc-out/> folder contains:
+
+\begin{itemize}
+\item \<build\_output.txt>: Logs collected from compiling the target project
```

Comment:
I'd add "without the Checker Framework" or similar to the end of this sentence, to make it clear that this file is about the target project exclusively. If there is a problem here, the rest of the script can't possibly succeed.